api/v1beta1/gcpmachine_types.go

```diff
@@ -66,6 +66,62 @@ const (
 	IPForwardingDisabled IPForwarding = "Disabled"
 )
 
+type SecureBootPolicy string
+
+const (
+	// SecureBootPolicyEnable enables the secure boot configuration for the GCP machine.
+	SecureBootPolicyEnable SecureBootPolicy = "Enabled"
+	// SecureBootPolicyDisable disables the secure boot configuration for the GCP machine.
+	SecureBootPolicyDisable SecureBootPolicy = "Disable"
```

Comment:
Should this be `Disabled` to match the `Enabled` semantic?

Reply:
Yes, updating